### ISSUE TITLE
Running with no parameters outputs '### Performing 2nd pass...'

### ISSUE THREAD

**Issue by mungojam** [hvanbakel/CsprojToVs2017#163]
A couple of things here:

> C:\Users\mark\source\repos\Force.com-Toolkit-for-NET>csproj-to-2017
### Performing 2nd pass to analyze converted projects...
> C:\Users\mark\source\repos\Force.com-Toolkit-for-NET>csproj-to-2017
### Performing 2nd pass to analyze converted projects...
> C:\Users\mark\source\repos\Force.com-Toolkit-for-NET>csproj-to-2017
> C:\Users\mark\source\repos\Force.com-Toolkit-for-NET>csproj-to-2017

1. It outputs the message when it should just output the usage instructions or an error or something
2. It only sometimes outputs it, which makes me wonder if we need an extra flush or something

**Comment by andrew-boyarshin** [comment]
Overall the fast console output on .NET Framework is unstable (line contents are mixed even when flushing output stream). On Mono (Windows too) no issues were detected. I hope that in v2 new logging will fix these issues.

**Comment by mungojam** [comment]
This was with the global tool, which is .net core 2.1, do you think that is still affected with the instabilities? It still needs a fix to the fact that it writes out to the log unconditionally.

**Comment by andrew-boyarshin** [comment]
v2 greatly reduces log noise (FileTransformation was irritating and useless) and introduces new logging system, so I believe issues with mixed line contents will be fixed.
As far as I understand the issue is the output of `### Performing 2nd pass to analyze converted projects...` when no options are specified. That's weird and I will have to do additional tests for that. I was under impression _CommandLine_ library takes care of such things.

**Comment by mungojam** [comment]
> I was under impression CommandLine library takes care of such things.

Perhaps it does actually, I've just tried it a few times with the newer version from your branch and it doesn't do it anymore. I had just seen the code and it looked like it was writing the message unconditionally but I guess I was missing something.

**Comment by hvanbakel** [comment]
Do we actually want to fix something here? I'm lost to what the 'issue' here is.

**Comment by andrew-boyarshin** [comment]
When no options specified - no output (expected help or at least any reasonable message).
In earlier versions `### Performing 2nd pass to analyze converted projects...` was printed.
It means `CommandLine` calls worker routine in `Program` even when nothing is specified on the command line.

There also was an issue with console output buffer (presumably fixed by migrating to `Microsoft.Extensions.Logging`). Lines were mixed or omitted in the output on .NET Framework/.NET Core (simple Flush calls I've inserted a long time ago didn't help).

**Comment by andrew-boyarshin** [comment]
No, the first paragraph still applies even to the 3.0.0 version.
